components/arm/universalrobots/ur.go

```diff
@@ -479,6 +480,9 @@ func (ua *URArm) CurrentInputs(ctx context.Context) ([]referenceframe.Input, err
 
 // GoToInputs TODO.
 func (ua *URArm) GoToInputs(ctx context.Context, goal []referenceframe.Input) error {
+	if ctx.Err() != nil {
```

Comment:
This is where we would catch a cancelled context that was given during planning.

Reply:
This addition is probably not necessary, as it will be caught in the move to joint positions call that we are proxying.

Reply:
Agreed. I think we should check for cancelled context before each call to `ua.connControl.Write` in `MoveToJointPositionRadians`